portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -1327,9 +1327,69 @@ definition {
 	@ignore = "true"
 	@priority = "3"
 	test CloseContentPerformancePanel {
```

Comment:
In some places you call it Content Performance panel, in others you talk about Metrics. I think for the Conten Dashboard it is better if you call it **Metrics**, since the button is called like this.

Reply:
Done